gsa/src/web/components/menu/__tests__/usermenu.js

```diff
@@ -81,46 +75,71 @@ describe('UserMenu component tests', () => {
   });
 
   test('should logout user on click', async () => {
+    const mock = {
+      request: {
+        query: LOGOUT,
+        variables: {},
+      },
+      result: {
+        data: {
+          logout: {
+            ok: true,
+          },
+        },
+      },
+    };
     const doLogout = jest.fn().mockResolvedValue();
     const gmp = {
       settings: {
         enableHyperionOnly: false,
       },
       doLogout,
     };
-    const {render} = rendererWith({gmp, store: true, router: true});
+    const {render} = rendererWith({
+      gmp,
+      store: true,
+      router: true,
+      queryMocks: [mock],
+    });
 
-    const {getByTestId} = render(
-      <MockedProvider mocks={mocks} addTypename={false}>
-        <UserMenu />
-      </MockedProvider>,
-    );
+    const {getByTestId} = render(<UserMenu />);
 
     const userSettingsElement = getByTestId('usermenu-logout');
 
     fireEvent.click(userSettingsElement);
+
     await wait();
 
     expect(gmp.doLogout).toHaveBeenCalled();
   });
 
   test('should renew session timeout on click', () => {
-    const renewSession = jest
-      .fn()
-      .mockResolvedValue({data: '2019-10-10T12:00:00Z'});
+    const renewDate = '2019-10-10T12:00:00Z';
+    const [queryMock, resultFunc] = createRenewSessionQueryMock(renewDate);
+
+    const renewSession = jest.fn().mockResolvedValue({data: renewDate});
     const gmp = {
       user: {
         renewSession,
       },
+      settings: {
+        isHyperionOnly: false,
```

Comment:
I think this is `enableHyperionOnly` in the gmpsettings

Reply:
Ups. Should be fixed now. Wasn't the only line...